### ISSUE TITLE
what is the parameters to balance cross-entropy loss and total cost

### ISSUE THREAD

**Issue by lxiongh** [facebookresearch/mobile-vision#23]
The total training loss consists of (1) cross-entropy loss and (2) total cost, which is the sum of cost from all layers: costtotal = Σlcostl.

How to balance
